packages/terra-heading/src/terra-dev-site/doc/example/HeadingWeights.jsx

```diff
@@ -3,6 +3,12 @@ import Heading from 'terra-heading';
 
 const HeadingWeights = () => (
   <div>
+    <p>
```

Comment:
Don't put documentation inside the examples, move this it the MDX documentation above the example.